controllers/repository_test.go

```diff
@@ -122,3 +123,118 @@ func TestListRepositories(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestGetImage(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockECR := mock.NewMockECRAPI(ctrl)
+	controller := NewRepositoryController(mockECR)
+
+	validateDescribeImagesInput := func(input *ecr.DescribeImagesInput) {
+		if v, want := aws.StringValue(input.RepositoryName), "user/test"; v != want {
+			t.Errorf("Name was '%v', expected '%v'", v, want)
+		}
+		if v, want := aws.StringValue(input.ImageIds[0].ImageTag), "test"; v != want {
+			t.Errorf("Tag was '%v', expected '%v'", v, want)
+		}
+	}
+
+	tag := ""
+
+	tags := []*string{}
+	tags = append(tags, &tag)
+
+	detail := &ecr.ImageDetail{}
+	detail.SetImageDigest("")
+	detail.SetImagePushedAt(time.Time{})
+	detail.SetImageSizeInBytes(0)
+	detail.SetImageTags(tags)
+	detail.SetRepositoryName("")
+
+	output := &ecr.DescribeImagesOutput{}
+	output.SetImageDetails([]*ecr.ImageDetail{detail})
+
+	mockECR.EXPECT().
+		DescribeImages(gomock.Any()).
+		Do(validateDescribeImagesInput).
+		Return(output, nil)
+
+	c := generateContext(t, nil, map[string]string{"tag": "test", "name": "test", "owner": "user"})
+	if _, err := controller.GetImage(c); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestDeleteImage(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockECR := mock.NewMockECRAPI(ctrl)
+	controller := NewRepositoryController(mockECR)
+
+	validateBatchDeleteImageInput := func(input *ecr.BatchDeleteImageInput) {
+		if v, want := aws.StringValue(input.RepositoryName), "user/test"; v != want {
+			t.Errorf("Name was '%v', expected '%v'", v, want)
+		}
+
+		if v, want := aws.StringValue(input.ImageIds[0].ImageTag), "test"; v != want {
+			t.Errorf("Tag was '%v', expected '%v'", v, want)
+		}
+	}
+
+	output := &ecr.BatchDeleteImageOutput{}
```

Comment:
[nit] no need to declare this variable here, can just put it into `Return(...)` function